content/en/05-reference/01-admin/07-debug-symbols.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Debug Symbols"
+metaTitle: "Reference | Admin | Debug Symbols"
+metaDescription: "Check for debug symbols."
+order: 7
+---
+
+Pixie automatically traces requests of a variety of protocol types. A subset of the supported protocols and encryption libraries require debug and/or DWARF symbols to work with Pixie.
+
+## Checking for debug symbols
+
+You can verify that an executable has debug symbols using `nm`:
+
+```bash
+nm <executable>
+```
+
+An output of `no symbols` means the application has no debug symbols and won't be traced.
+
+## Checking for DWARF symbols
```

Comment:
DWARF symbols -> DWARF debug information

Same changes in the text for consistency.